source/content/migrate-manual.md

```diff
@@ -114,7 +114,7 @@ Your **code** is all custom and contributed modules or plugins, themes, and libr
 The codebase for each CMS upstream offered by Pantheon can be found on GitHub:
 
  - [Drupal 7](https://github.com/pantheon-systems/drops-7)
- - [Drupal 8](https://github.com/pantheon-systems/drops-8)
+ - [Drupal 9+](https://github.com/pantheon-systems/drupal-composer-managed)
```

Comment:
@greg-1-anderson, is there a different repo for D10?

Reply:
What's the difference between 

https://github.com/pantheon-upstreams/drupal-composer-managed

and https://github.com/pantheon-upstreams/drupal-10-composer-managed

Reply:
Drupal 10 composer managed is a new class of upstreams called a "start state" that applies some different composer configuration and then sets them back on drupal-composer-managed

Reply:
> a new class of upstreams called a "start state"

@greg-1-anderson do we have a write up somewhere that describes this distinction?

Reply:
Does this mean that Drupal 8 is end-of-sale? (Are we going to take active measures to prevent or restrict new sites from coming on the platform?)

I do think that we should list D9 and D10 separately here (and D11, once available). I don't know if there is a docs page that talks about start state upstreams; we should write one if there isn't.